__tests__/api.test.ts

```diff
@@ -39,15 +69,16 @@ test('API: get available translations correctly gets all locales', async () => {
     expect(locales).toEqual(expectedLocales.join(', '));
 });
 
-test('API: read file->translate->apply->write', async () => {
+test.skip('API: read file->translate->apply->write', async () => {
```

Comment:
Why is this skipped? Did something break with the read, translate, apply write?

Reply:
These tests were initially skipped, probably because they don't mock real Azure call. There was a dummy test "test.only('API: tests are currently ignored - comment out this test to run actual tests.')". I can't use same technic to ignore some tests, because I added some real tests to this file.